src/com/google/javascript/jscomp/js/es6/util/inherits.js

```diff
@@ -58,14 +58,17 @@ $jscomp.inherits = function(childCtor, parentCtor) {
   /** @override */
     childCtor.prototype.constructor = childCtor;
 
-  for (var p in parentCtor) {
-    if (Object.defineProperties) {
-      var descriptor = Object.getOwnPropertyDescriptor(parentCtor, p);
-      if (descriptor) {
-        Object.defineProperty(childCtor, p, descriptor);
+  if (Object.defineProperties) {
+    var propNames = Object.getOwnPropertyNames(parentCtor);
+    for (var i = 0; i < propNames.length; i++) {
+      var descriptor = Object.getOwnPropertyDescriptor(parentCtor, propNames[i]);
```

Comment:
@brad4d My understanding is the require statements are needed for methods which you can polyfill. However there is no polyfill for `getOwnPropertyNames` for ES3. So in this case I didn't import them. `Object.defineProperties` is the same way - no ES3 equivalent.